src/test/java/uk/gov/companieshouse/items/orders/api/service/CertificateItemServiceUnitTest.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.companieshouse.items.orders.api.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.items.orders.api.model.CertificateItem;
+import uk.gov.companieshouse.items.orders.api.model.Item;
+import uk.gov.companieshouse.items.orders.api.repository.CertificateItemRepository;
+
+import java.time.LocalDateTime;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+/**
+ * Unit tests the {@link CertificateItemService} class.
+ */
+@ExtendWith(MockitoExtension.class)
+class CertificateItemServiceUnitTest {
```

Comment:
We are [following Google's Java coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md). "Unit" won't be included in the test class name. See https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names.

Reply:
Technically speaking, that standard doesn't preclude the use of the `...UnitTest` ending for a class name. However, I am quite happy to call all of the tests `..Test`.

Reply:
...Test would be consistent with our other repos